ocamlformat-help.txt

```diff
@@ -9,140 +9,125 @@ DESCRIPTION
 
 OPTIONS (CODE FORMATTING STYLE)
        Unless otherwise noted, any option --option=VAL detailed in this
-       section can be set in an .ocamlformat configuration file with an
-       'option = VAL' line, or with an [@@@ocamlformat "option=VAL"]
-       attribute.
-
-       --break-cases={fit|nested|all} (absent=fit or OCAMLFORMAT_BREAK_CASES
-       env)
+       section can be set in many ways, its value is determined in the
+       following order (of increasing priority): it can be set in an
+       .ocamlformat configuration file with an 'option = VAL' line, or using
```

Comment:
Should we mention that .ocamlformat files in current and all ancestor directories of the input file are used, with more distant ancestors having lower priority?